Sources/RichTextKit/RichTextContext.swift

```diff
@@ -8,6 +8,10 @@
 
 import SwiftUI
 
+protocol RichTextContextDelegate: AnyObject {
```

Comment:
Thank you for contributing with these fixes!

I'd like to keep PRs atomic, and for this PR to just contain the subscription fix. Can you move this to the PR for updating the text, together with the code I've commented on below?